tools/sigclient/functionalQueries/functionalQueries.yml

```diff
@@ -35,19 +35,19 @@ sort_tail_eval_newField.json
 tail_stats.json
 dedup_sortby_consecutive.json
 dedup_sortby.json
+streamstats_win3.json
+streamstats_win4_currF.json
+streamstats_win4_grpby.json
+streamstats_win4_reset_before_after.json
+streamstats_win2_gloF_currF_grpby.json
+streamstats_win2_gloF_grpby.json
+streamstats_multi_resets_grpby.json
+streamstats_eval.json
+streamstats_after_stats.json
+streamstats_after_grpby.json
 
```

Comment:
do consider adding results for `streamstats_without_sort.json` for both functional and longerFunctional